build.gradle

```diff
@@ -22,85 +21,131 @@ group 'io.xh'
 apply plugin:'idea'
 apply plugin:'war'
 apply plugin:'org.grails.grails-web'
-apply plugin:'org.grails.grails-gsp'
 
 repositories {
     mavenLocal()
-    mavenCentral()
     maven {url 'https://repo.grails.org/grails/core'}
     maven {url 'https://repo.xh.io/content/groups/public/'}
 }
 
-configurations.all {
-    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
```

Comment:
This gave us the behavior of looking for new snapshot versions each time dependencies were resolved.
Looks like the default [remains 24 hours](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ResolutionStrategy.html#org.gradle.api.artifacts.ResolutionStrategy:cacheChangingModulesFor(int,%20java.util.concurrent.TimeUnit)), unless there is some other way this is being set, which might be unexpectedly long when using any snapshot builds.

Reply:
will restore this -- great catch.